lib/node_modules/@stdlib/blas/base/ssbmv/test/fixtures/julia/strides_xoyt.json

```diff
@@ -0,0 +1,15 @@
+{
+  "uplo": "upper",
+  "order": "row-major",
+  "N": 3,
+  "k": 1,
+  "alpha": 2.0,
+  "beta": 1.0,
+  "lda": 3,
+  "strideX": 1,
+  "strideY": 2,
+  "A": [ 0.5, -0.2, 0.1, 0.4, 0.6, -0.3, 0.7, 0.2, -0.8 ],
+  "x": [ 1.0, 2.0, 3.0 ],
+  "y": [ 1.0, 2.0, 3.0 ],
+  "y_out": [ 1.2, 2.0, 7.8 ]
```

Comment:
This is incorrect and was also present in the other PRs. Namely, your `y` stride is `2`. These arrays cannot accommodate a stride of `2`, given that `N` is `3`. These should be `6` element arrays and your expected values should be updated accordingly.

Reply:
In general, this has been a recurring PR review comment. In short, anytime you have non-unitary strides, you need to more carefully consider the corresponding buffer implications.

Reply:
Sure, @kgryte I think I missed that. I will keep this in mind.